src/main/webapp/index.html

```diff
@@ -1,12 +1 @@
-<!DOCTYPE html>
-<html>
-  <head>
-    <meta charset="UTF-8">
-    <title>Nina's Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
-  </head>
-  <body>
-      <p> Hello </p>
-  </body>
-</html>
+
```

Comment:
Do you want to just delete this file instead?

Reply:
I think if I delete index.html,  I get an error when running my dev server because it doesn't know where to go upon startup. I usually just have a blank file for this and then navigate to search.html